src/googledrive.js

```diff
@@ -114,13 +142,22 @@ GoogleDrive.prototype = {
   connected: false,
   online: true,
 
+  /**
+   * Configure the Google Drive backend.
+   *
+   * Fetches the user info from Google when no ``userAddress`` is given.
+   *
+   * @param {object} settings
+   * @param {string} [settings.userAddress] - The user's email address
+   * @param {string} [settings.token] - Authorization token
+   */
```

Comment:
I think this is a private method, because a developer should only use `connect()` and have the library deal with the rest, no?